code/language/c/loop/While.c

```diff
@@ -0,0 +1,22 @@
+/*While loop is another loop in C language.*/
```

Comment:
A while loop in C programming repeatedly executes a target statement as long as a given condition is true. This sounds better than saying it is another loop. Another loop sounds informal.